lib/react-permissible.js

```diff
@@ -1 +1,6 @@
 "+o+" `"+u+"` supplied to `"+r+"`.")}if(!Array.isArray(t))return"production"!==e.env.NODE_ENV&&a("Invalid argument supplied to oneOfType, expected an instance of array."),r;for(var o=0;o<t.length;o++){var u=t[o];if("function"!=typeof u)return a("Invalid argument supplied to oneOfType. Expected an array of check functions, but received "+j(u)+" at index "+o+"."),r}return p(n)}function _(t){function e(e,n,r,o,u){var a=e[n],c=O(a);if("object"!==c)return new s("Invalid "+o+" `"+u+"` of type `"+c+"` supplied to `"+r+"`, expected `object`.");for(var f in t){var p=t[f];if(p){var l=p(a,f,r,o,u+"."+f,i);if(l)return l}}return null}return p(e)}function m(t){function e(e,n,r,u,a){var c=e[n],f=O(c);if("object"!==f)return new s("Invalid "+u+" `"+a+"` of type `"+f+"` supplied to `"+r+"`, expected `object`.");var p=o({},e[n],t);for(var l in p){var y=t[l];if(!y)return new s("Invalid "+u+" `"+a+"` key `"+l+"` supplied to `"+r+"`.\nBad object: "+JSON.stringify(e[n],null,"  ")+"\nValid keys: "+JSON.stringify(Object.keys(t),null,"  "));var v=y(c,l,r,u,a+"."+l,i);if(v)return v}return null}return p(e)}function g(e){switch(typeof e){case"number":case"string":case"undefined":return!0;case"boolean":return!e;case"object":if(Array.isArray(e))return e.every(g);if(null===e||t(e))return!0;var n=c(e);if(!n)return!1;var r,o=n.call(e);if(n!==e.entries){for(;!(r=o.next()).done;)if(!g(r.value))return!1}else for(;!(r=o.next()).done;){var i=r.value;if(i&&!g(i[1]))return!1}return!0;default:return!1}}function x(t,e){return"symbol"===t||("Symbol"===e["@@toStringTag"]||"function"==typeof Symbol&&e instanceof Symbol)}function O(t){var e=typeof t;return Array.isArray(t)?"array":t instanceof RegExp?"object":x(e,t)?"symbol":e}function w(t){if(void 0===t||null===t)return""+t;var e=O(t);if("object"===e){if(t instanceof Date)return"date";if(t instanceof RegExp)return"regexp"}return e}function j(t){var e=w(t);switch(e){case"array":case"object":return"an "+e;case"boolean":case"date":case"regexp":return"a "+e;default:return e}}function P(t){return t.constructor&&t.constructor.name?t.constructor.name:k}var E="function"==typeof Symbol&&Symbol.iterator,T="@@iterator",k="<<anonymous>>",S={array:l("array"),bool:l("boolean"),func:l("function"),number:l("number"),object:l("object"),string:l("string"),symbol:l("symbol"),any:function(){return p(r)}(),arrayOf:y,element:function(){function e(e,n,r,o,i){var u=e[n];if(!t(u)){return new s("Invalid "+o+" `"+i+"` of type `"+O(u)+"` supplied to `"+r+"`, expected a single ReactElement.")}return null}return p(e)}(),instanceOf:v,node:function(){function t(t,e,n,r,o){return g(t[e])?null:new s("Invalid "+r+" `"+o+"` supplied to `"+n+"`, expected a ReactNode.")}return p(t)}(),objectOf:h,oneOf:d,oneOfType:b,shape:_,exact:m};return s.prototype=Error.prototype,S.checkPropTypes=u,S.PropTypes=S,S}}).call(e,n(3))},function(t,e,n){"use strict";function r(t){if(null===t||void 0===t)throw new TypeError("Object.assign cannot be called with null or undefined");return Object(t)}/*
```

Comment:
proposition for further refactor:

this file should be automatically generated before publishing in CircleCI

CC @mdrazyk

Reply:
Agreed.